src/App.jsx

```diff
@@ -1,87 +1,82 @@
-import React from 'react';
+import React, { useState, useMemo, useEffect, useCallback } from 'react';
+import TodoApp from './components/TodoApp/TodoApp';
+import TodoList from './components/TodoList/TodoList';
+import TodoFilter from './components/TodosFilter/TodosFilter';
+import { TodoContext } from './TodoContext';
 
 function App() {
-  return (
-    <section className="todoapp">
-      <header className="header">
-        <h1>todos</h1>
+  const [inputText, setInputText] = useState('');
```

Comment:
inputText is used only inside TodoApp, so actually I don't see a reason to store this state inside App and then pass it to context, just make it TodoApp's state and use it there, without context